src/Sulu/Component/Util/SuluNodeHelper.php

```diff
@@ -164,7 +164,7 @@ public function hasSuluNodeType($node, $suluNodeTypes)
      */
     public function extractWebspaceFromPath($path)
     {
-        $match = preg_match('/^\/' . $this->getPath('base') . '\/(\w*)\/.*$/', $path, $matches);
+        $match = preg_match('/^\/' . $this->getPath('base') . '\/([^\/]*)\/.*$/', $path, $matches);
```

Comment:
Ok, looks like PHPCR somehow allows whitespaces in node names, but Sulu is not working correctly then, there are quite some requests failing.

But I think this is the wrong place to restrict that. I would prefer to leave the regex here like this, because I think it is not necessary that this component knows about it. The regex like this is the safest option.

Instead I would add  a validation for the webspace key in the XSD or the `XmlFileLoader` loading the XML. Otherwise the system would allow the "invalid" key to pass, and this method would not fail, but return wrong data.